cmd/prow-job-dispatcher/main.go

```diff
@@ -370,4 +424,61 @@ func main() {
 	if err := dispatcher.SaveConfig(config, o.configPath); err != nil {
 		logrus.WithError(err).Fatalf("Failed to save config file to %s", o.configPath)
 	}
+
+	if !o.createPR {
+		logrus.Info("Finished dispatching and create no PR, exiting ...")
+		os.Exit(0)
+	}
+
+	logrus.Infof("Changing working directory to '%s' ...", o.targetDir)
```

Comment:
Please do not `printf` logging, use fields

Reply:
Updated.